lib-database-entities/src/commonMain/kotlin/com/ustadmobile/lib/db/entities/StatementEntity.kt

```diff
@@ -97,6 +99,11 @@ open class StatementEntity {
 
     var extensionProgress: Int = 0
 
+    /**
+     *  indicates whether or not the statement is about the root contentEntry or child entries
```

Comment:
Let's add to the comment:

This is used by queries (e.g. for reports)  e.g. to see if a "completed" verb applies to the contententry itself, or only a subsection (child) of the content